lib/manageiq/api/client/mixins/custom_inspect_mixin.rb

```diff
@@ -0,0 +1,30 @@
+module CustomInspectMixin
+  extend ActiveSupport::Concern
+
+  DEFAULT_CUSTOM_INSPECT_EXCLUSIONS = [:@custom_inspect_exclusions].freeze
+
+  def inspect
+    res = "#<#{self.class.name}:#{object_id} "
```

Comment:
This is not correct (object_id is not what is normally shown).  There is a neat little trick you can do instead:

```ruby
Kernel.instance_method(:to_s).bind(self).call.chomp!('>')
```

You can see how azure-armrest gem does this as an example in its own inspect override: https://github.com/ManageIQ/azure-armrest/blob/master/lib/azure/armrest/model/base_model.rb#L97-L102